### PR TITLE
EXP-164 Add css icons to grpc response and update proto

### PR DESCRIPTION
**Pull request by 0xdev22**
# Proposed Changes
The change adds css icons as part of the response from grpc for features compatibility

### CODE REVIEW

File: pkg/grpc/device_definition.proto [hunk 1]
```diff
@@ -250,6 +250,7 @@ message GetDeviceCompatibilityListRequest {
 message Feature {
 	string key = 1;
 	int32 support_level = 2;
+	string css_icon = 3;
```
Comment (elffjs) [inline pkg/grpc/device_definition.proto:253]: One thing to consider is that the CSS icon is optional in the database. Should be optional in the message here? We could stick with empty, just a question.

File: internal/core/queries/device_compatibility.go [hunk 2]
```diff
@@ -42,9 +42,12 @@ func (dc GetDeviceCompatibilityQueryHandler) Handle(ctx context.Context, query m
...
 	}
 
-	integFeats := make(map[string]string, len(inf))
+	integFeats := make(map[string]map[string]string, len(inf))
 	for _, k := range inf {
-		integFeats[k.FeatureKey] = k.DisplayName
+		integFeats[k.FeatureKey] = map[string]string{
+			"displayName": k.DisplayName,
+			"cssIcon":     k.CSSIcon.String,
+		}
```
Comment (elffjs) [inline internal/core/queries/device_compatibility.go:50]: Maybe make a little struct for the value here?

File: pkg/grpc/device_definition.proto (hunk 1 above)
Comment (JamesReate) [reply pkg/grpc/device_definition.proto:253]: would that be an optional using something like this? https://pkg.go.dev/github.com/gogo/protobuf/gogoproto#hdr-More_Canonical_Go_Structures

Comment (elffjs) [reply pkg/grpc/device_definition.proto:253]: Can we leave a comment, at least? "May be blank"?

**Pull request merged**